### DISCUSSION TITLE
Default radio cover art causes chromium to use 20% CPU

### DISCUSSION THREAD

**Discussion by vale-max** [rern/rAudio#342] [Fixed Issues]
Not sure if really an issue, but on Raspberry 4 local browser the default animated VU meter cover art makes chromium use a lot of CPU

  19983 root      20   0   37.1g 137184  87516 S  13.6   7.1   6:11.24 chromium
  20031 root      20   0  788592  79944  48556 S  11.3   4.1   4:43.12 chromium

**Comment by AndyFx12** [comment]
Try rAudio-Logo (see Playback settings) for "No cover".

Andy

**Comment by rern** [comment]
Chromium browser on RPi is not good at rendering animation. A simple blinking icon could consume 50% or more of CPU cycles. The time knob progress on local screen is not as smooth as on remote screen because it's refreshed once a second to reduce the CPU load.

**Reply by vale-max** [reply]
I noticed that the inteface is not terribly friendly in the 3.5" tft. It is easy to touch the wrong part of the image to stop or go back/fwd and yes, the volume knob is not very responsive.
Would be nice to have a simplified skin with big always visible buttons and maybe a linear volume slider.
